### ISSUE TITLE
installation py.test error: unrecognized arguments

### ISSUE THREAD

**Issue by jkikstra** [iiasa/ixmp#193]
When installing `ixmp` from the source code (following https://message.iiasa.ac.at/projects/ixmp/en/stable/install.html) and run `py.test`, I get the following error message:
```
C:\Users\kikstra\Documents\GitHub\ixmp>py.test
C:\Users\kikstra\AppData\Local\Continuum\anaconda3\lib\site-packages\xarray\core\merge.py:10: FutureWarning: The Panel class is removed from pandas. Accessing it from the top-level namespace will also be removed in the next version
  PANDAS_TYPES = (pd.Series, pd.DataFrame, pd.Panel)
ERROR: usage: py.test [options] [file_or_dir] [file_or_dir] [...]
py.test: error: unrecognized arguments: --cov=ixmp --cov-config=ci/coveragerc --cov-report --cov-report xml
  inifile: C:\Users\kikstra\Documents\GitHub\ixmp\setup.cfg
  rootdir: C:\Users\kikstra\Documents\GitHub\ixmp
```

**Comment by khaeru** [comment]
Good catch. `pytest-cov` should also be among the dependencies installed by `pip install .[tests]`, but is not:
https://github.com/iiasa/ixmp/blob/5066096164c779c5bbfa60dbbc8ec73fcbda55a2/setup.py#L26-L30

**Comment by khaeru** [comment]
Closed by @zikolach in #207.
